edm.yaml

```diff
@@ -192,21 +192,41 @@ datatypes :
      - fcc::CaloHit hits // Hits used in the cluster
 
   fcc::TrackState:
-    Description: "Track state at a given point on the track."
+    Description: "Track state at a given point on the track. The track parameters are chosen 
+    to be decribed in the curvilinear frame of the track. With the track parameters 
+    (local1:=0,local2:=0,phi,theta,q/p) defined at the origin of a surface perpendicular to 
+    the track. The local coordinates are zero per defintion. The azimuthal and polar angle
+    can be calculated from the direction of the momentum. The last parameter q/p can be 
+    calculated from the charge and the value of the momentum."
     Author : "C. Bernet, B. Hegner"
     Members:
-     - float location // Location on the track. (Radius?)
-     - float omega // Track curvature in cm.
+     - fcc::Point position // global position specifying the location on the track  
+     - float[3] momentum   // the momentum at position  
+     - float charge        // the charge
+     - float[15] cov       // the covariance matrix describing the covariances of the five 
+       		           // track parameters. Duplicated entries have been left out, which 
+		           // reduces thesize to 15 entries      
+
+  fcc::PerigeeTrackState:
```

Comment:
Yes, this is what I was writing I guess. 
So to describe a state, we need: 

- fcc::Point : reference point
- d0
- z0
- qOverp
- phi
- theta

I do think that we should use a single class, fcc::TrackState, with these data members 
(please drop PerigeeTrackState).  

```yaml
  fcc::TrackState:
    Description: "Track state."
    Author : "J. Hrdinka"
    Members:
     - float phi // azimuthal angle describing the direction of the momentum
     - float theta // polar angle describing the direction of the momentum
     - float qOverP // describing the curvature, bending direction and momentum 
     - float d0 // Transverse impact parameter
     - float z0 // Longitudinal impact parameter
     - fcc::Point referencePoint // Point from which d0, z0 are measured
     - float[15] cov       // the covariance matrix describing the covariances of the five 
       		                       // track parameters. Duplicated entries have been left out, which 
                                       // reduces thesize to 15 entries      
```

Then: 

```yaml 
 fcc::Track:
    Description: "Track reconstructed from clusters in the inner tracker"
    Author : "C. Bernet, B. Hegner"
    Members:     
     - float chi2 // chi2 returned by the track fit
     - unsigned ndf // Number of degrees of freedom of the track fit
     - unsigned bits // Stores flags
    OneToManyRelations :
     - fcc::TrackCluster clusters // Clusters used for reconstruction
     - fcc::TrackState states // states along the track. The first state is taken at the point of closest approach.
```

Would that be ok for you? 

I'm a bit disturbed by one last thing, you might want to discuss it with Andi. 
In this alternative PR, PR #33, we propose an equivalent parametrization, the one used by the ILC/CLIC guys. 
In their document http://www-flc.desy.de/lcnotes/notes/LC-DET-2006-004.pdf they say that their parametrization reduces the need for reinterpretation of the parameters. I think we should give a bit of thinking about that. For example, tan lambda instead of theta could be wise, I really don't know, that's you call.